cmake/modules/FindWine.cmake

```diff
@@ -83,7 +87,10 @@ IF(WINE_CXX)
 ENDIF()
 
 FIND_PATH(WINE_INCLUDE_DIR wine/exception.h
-	HINTS ${WINE_INCLUDE_HINT}
+	HINTS ${WINE_INCLUDE_HINT} 
+	/usr/bin/wine/ 
+	/opt/wine-staging/lib/wine/i386-unix
```

Comment:
This seems oddly specific... Perhaps a more exhaustive listing?  (Not recommending it in completion, just as a pointer).

https://github.com/wineasio/wineasio/blob/652964155dcee005078c7cb652656e8a0b995186/wineasio-register#L14-L31

Reply:
Actually, scratch that.  Best to not blindly crawl for unrelated wine versions on the system.